.buildkite/scripts/exhaustive-tests/generate-steps.py

```diff
@@ -48,6 +48,7 @@ def compat_step(imagesuffix: str, command: LiteralScalarString) -> dict[str, typ
         "key": slugify_bk_key(f"compat-linux-{imagesuffix}"),
         "command": command,
         "agents": {},
+        "retry": {"automatic": [{"limit": 3}]},
```

Comment:
this is the same thing done in https://github.com/elastic/logstash/pull/15874/files#diff-28f034d735238791a51964b6bb47e1f6579001369bc2b569d837449ab710b5f8R145